src/main/java/org/folio/consortia/service/impl/PermissionUserServiceImpl.java

```diff
@@ -70,6 +70,12 @@ public void addPermissions(PermissionUser permissionUser, String permissionsFile
     });
   }
 
+  @Override
+  public void deletePermissionUser(String userId) {
+    permissionsClient.deletePermissionUser(userId);
+    log.info("deleteUserPermissions:: Deleted permissionUser with userId={}", userId);
```

Comment:
I propose to move log.info above by correcting method name in logs.  
log.info("deletePermissionUser:: Deleting permissionUser with userId={}", userId);
permissionsClient.deletePermissionUser(userId); 

In this case its easy to debug if deletingPermissionUser failed.

Reply:
i thought, if client throw error, we didn't see deleted permission log. we will see if operation is success, we don't see if operation fail

Reply:
Btw, thanks to your suggestion, I will add extra log in userTenantService part